source/javascripts/meetups.js

```diff
@@ -45,7 +45,9 @@
 
   handler.buildMap(mapOptions, function() {
     if(navigator.geolocation) {
-      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap);
+      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap, function(){
```

Comment:
The 2nd callback is the error handler which handles the case where the user doesn't permit the geolocation API to have access to her current location. I guess it might make more sense to insert all the markers first, before calling the geolocation API, and then just change the extent of the existing map in the callback. I will put together another commit for this later today